Automattic/jetpack
# Fix | Publicize remains ON after the post is published

After #34064, there is some regression that resulted in publicize remaining ON even after the post is published. The reason for that is when a post is published, the API call returns `jetpack_publicize_connections` in the post meta with `enabled` set to `false` for all the connections. But, in #34064, we changed the source of truth for connections from post meta to social store thus the issue.

## Proposed changes:
* Sync the connections from post attributes to the social store after a post is published

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* For a site with Jetpack Social enabled, add some social connections
* Create a new post
* Publish the post 
* Open the Jetpack/Social sidebar
* Confirm that publicize and all the connections are disabled

<img width="299" alt="Screenshot 2023-11-24 at 2 24 52 PM" src="https://github.com/Automattic/jetpack/assets/18226415/808217d8-80fc-4076-8734-494934971296">
